tox.ini

```diff
@@ -201,13 +201,13 @@ commands = python scripts/release.py {posargs}
 
 [pytest]
 minversion = 2.0
-plugins = pytester
-addopts = -ra -p pytester --ignore=testing/cx_freeze
```

Comment:
no cx_freeze anymore?

Reply:
Oh it is OK, we have an `freeze` testenv dedicated to it. 👍

Reply:
@nicoddemus but we no longer ignore it now

Reply:
That's OK, it contains only two dummy tests: https://github.com/pytest-dev/pytest/tree/master/testing/freeze/tests

We could perhaps rename them to something else that's not picked up by pytest by default, but not sure it is worth the trouble. 😁

Reply:
AFAIK the `cx_freeze` gets generated now.
`cx_freeze` would not have matches `freeze` currently anyway.